src/sp/it/pl/gui/objects/autocomplete/AutoCompletePopup.kt

```diff
@@ -53,7 +52,7 @@ open class AutoCompletePopup<T>: PopupControl() {
         isAutoFix = true
         isAutoHide = true
         isHideOnEscape = true
-        styleClass += COMBO_BOX_STYLE_CLASS
+        styleClass += "combo-box-popup" // com.sun.javafx.scene.control.Properties.COMBO_BOX_STYLE_CLASS
```

Comment:
This was the original solution. I changed it exactly because I do not want to rely on values, even if they are sort of guaranteed to not change. I would prefer it to be reverted back.

Reply:
But by changing it like this, we depend on one added export less, the goal should be to get rid of them all. The constant is not actually gonna change ^^

Reply:
I doubt we will be able to remove all of these (I just added pushed add-exports to master haha, sorry), but we might as the use is always on the experimental side of things. Still while the goal is noble, it is of no importance per se.

What we should strive for is to be correct, not short or sloppy. Explicit and by the book. Random values in code are very bad and it does not matter if the chance of value changing is 99% or 0.0001%. It is simply wrong.

Removing the exports will not help anyone, in fact the opposite. If one day someone will need to add another one, he will have to research the syntax and pave the way again. Like with the local jars that we removed. Now we need it for latest jaudiotagger...

This needs to be a constant. And it should be documented.

Now perhaps what we can do is initialize the constant with new ComboBox().getSKin()...getStyleclass() of some kind, although I doubt it. In the worst case, the constant will be string literal, but I'd rather use the exported value, just to be sure. It is what we want.

Reply:
we are all so smart. There is `com.sun.javafx.scene.control.Properties.COMBO_BOX_STYLE_CLASS` freely accessible ^^

Reply:
I will think about this some more and consider using simple constant. The reason we may not want to use the inaccessible values is related to whether it is good to rely on the actual skin implementations.